GoogleTestAdapter/Core/GoogleTestExecutor.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Linq;
+﻿using System;
```

Comment:
Needs Microsoft-modification header.
Apply to all other modified files missing it as well.